dockers/docker-orchagent/copp.json.j2

```diff
@@ -0,0 +1,85 @@
+[
+    {
+        "COPP_TABLE:default": {
+            "queue": "0",
+            "meter_type":"packets",
+            "mode":"sr_tcm",
+            "cir":"600",
+            "cbs":"600",
+            "red_action":"drop"
+        },
+        "OP": "SET"
+    },
+    {
+        "COPP_TABLE:trap.group.bgp.lacp": {
+            "trap_ids": "bgp,bgpv6,lacp",
+            "trap_action":"trap",
+            "trap_priority":"4",
+            "queue": "4"
+        },
+        "OP": "SET"
+    },
+    {
+        "COPP_TABLE:trap.group.arp": {
+            "trap_ids": "arp_req,arp_resp,neigh_discovery",
+            "trap_action":"copy",
+            "trap_priority":"4",
+            "queue": "4",
+            "meter_type":"packets",
+            "mode":"sr_tcm",
+            "cir":"600",
+            "cbs":"600",
+            "red_action":"drop"
+        },
+        "OP": "SET"
+    },
+{% if DEVICE_METADATA['localhost']['type'] == "ToRRouter" %}
```

Comment:
It would be better to check for the existence of `type`. If not present, the template might crash - [Ref ](https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-orchagent/switch.json.j2#L4)

Reply:
Could you please ensure if the type is not defined, the trap is installed? Type is not a mandatory parameter and this might cause backward compatibility issues in cases where type field is not defined

Reply:
@prsunny @dgsudharsan Updated to add default else condition for backward compatibility 
if DEVICE_METADATA and it sub-parameters are not define.